test/TetherContent.spec.js

```diff
@@ -81,14 +81,14 @@ describe('TetherContent', () => {
   });
 
   describe('show', () => {
-    it('should be called on componentWillUnmount', () => {
+    it('should be called on componentDidMount', () => {
       state = true;
-      spyOn(TetherContent.prototype, 'componentWillUnmount').and.callThrough();
+      spyOn(TetherContent.prototype, 'componentDidMount').and.callThrough();
       spyOn(TetherContent.prototype, 'show').and.callThrough();
       const wrapper = mount(<TetherContent tether={tetherConfig} isOpen={state} toggle={toggle}><p>Content</p></TetherContent>);
       const instance = wrapper.instance();
 
-      expect(TetherContent.prototype.componentWillUnmount.calls.count()).toBe(0);
+      expect(TetherContent.prototype.componentDidMount.calls.count()).toBe(1);
```

Comment:
I threw this in here, the test didn't make any sense saying `show` should be called on unmount. Seemed like a copy paste from the one above it. Now this tests to make sure the show is called when the component did mount.